core/src/main/scala/org/apache/spark/sql/delta/OptimisticTransaction.scala

```diff
@@ -728,11 +728,12 @@ trait OptimisticTransactionImpl extends TransactionalWrite with SQLMetricsReport
       numCdcFiles = 0,
       cdcBytesNew = 0,
       protocol = postCommitSnapshot.protocol,
-      info = Option(commitInfo).map(_.copy(readVersion = None, isolationLevel = None)).orNull,
+      info = Option(commitInfo)
+        .map(_.copy(readVersion = None, isolationLevel = Option(isolationLevel.toString))).orNull,
```

Comment:
Since we have fixed the commit info, this can be `info = Option(commitInfo).map(_.copy(readVersion = None)).orNull`?

Reply:
Thanks for your feedback. Done.